### DISCUSSION TITLE
Any Donations?

### DISCUSSION THREAD

**Discussion by the-insulter** [joncage/ed-scout#180] [General]
Hi,

very nice too! and so slick. Thx.

You need donations?

ru

**Comment by joncage** [comment]
Need? No, I wrote this tool for fun and shared it because I found it invaluable.

If you want to show your appreciation though, I added a link to some options in the main readme:

> Enjoy this tool? Consider..
> 
> * [Donating to EDSM](https://www.edsm.net/en_GB/donation)
> * [Buying me a coffee](https://ko-fi.com/S6S8424WV)
> 
> ..to the scout can keep flying :-)
